collector/snapshots_reponse.go

```diff
@@ -43,6 +43,6 @@ type SnapshotStatDataResponse struct {
 
 // SnapshotRepositoriesResponse is a representation snapshots repositories
 type SnapshotRepositoriesResponse map[string]struct {
-	Type     string            `json:"type"`
-	Settings map[string]string `json:"settings"`
+	Type     string                 `json:"type"`
+	Settings map[string]interface{} `json:"settings"`
```

Comment:
if we don't use the value of settings anywhere, we could just remove the field 🤔

Reply:
> if we don't use the value of settings anywhere, we could just remove the field `Settings`
It seems the value of settings is not used now. I tried it  worked without `Settings` field, and it's another way to fix the bug.